src/OpenFeature.Contrib.ConfigCat/UserBuilder.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using ConfigCat.Client;
+using OpenFeature.Model;
+
+namespace OpenFeature.Contrib.ConfigCat
+{
+    internal static class UserBuilder
+    {
+        internal static User BuildUser(this EvaluationContext context)
+        {
+            if(context == null)
+            {
+                return null;
+            }
+
+            var user = new User(Guid.NewGuid().ToString());
```

Comment:
I'd only initialize the new User with a GUID if you could not parse the "id" or "identifier" from the context.

Reply:
Fair enough